developer/src/kmc-kmn/src/compiler/compiler.ts

```diff
@@ -42,50 +42,10 @@ const baseOptions: CompilerOptions = {
  */
 let callbackProcIdentifier = 0;
 
-/**
- * Pointer in wasm-space
- */
-type WasmPtr = number;
-
-/**
- * The wrapped functions
- */
-class WasmWrapper {
-  Module: any;
-
-  compileKeyboardFile?: (pszInfile: string, pszOutfile: string, aSaveDebug: number, aCompilerWarningsAsErrors: number, aWarnDeprecatedCode: number, msgProc: string) => boolean;
-  parseUnicodeSet?: (pat: string, buf: WasmPtr, length: number) => number;
-  setCompilerOptions?: (shouldAddCompilerVersion: number) => boolean;
```

Comment:
I see that `kmcmp_Wasm_SetCompilerOptions` and similar in CompilerInterfaces.cpp got marked deprecated, and that nomenclature seems pretty related to `setCompilerOptions` here.  I assume it's being fully removed here, hence why there's no equivalent after these changes?

Reply:
Yes, `compileKeyboardFile` and the stubbed and never-yet-interfaced-in-wasm `CompileKeyboardFileToBuffer`, togethjer with `setCompilerOptions` are replaced with `kmcmp_compile`. See #8870 for the completion of this refactoring.